WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -191,7 +191,8 @@ class MySiteViewModel @Inject constructor(
         isSiteSelected = false
         checkAndShowJetpackFullPluginInstallOnboarding()
         checkAndShowQuickStartNotice()
-//        bloggingPromptCardViewModelSlice.onResume(uiModel.value as? SiteSelected)
+        // bloggingPromptCardViewModelSlice.onResume(uiModel.value as? SiteSelected)
+        selectedSiteRepository.updateSiteSettingsIfNecessary()
```

Comment:
This line fixes the issue. We need to call this in order to initialise `siteSettings` in the selectedSiteRepository. In trunk, this is achieved by calling `mySiteSourceManager.onResume` inside `MySiteViewModel.onResume()`, but mySiteSourceManager was removed after the refactoring.

Reply:
Ohh okk got it 👍🏼  Good catch